api/openapi/auth.yml

```diff
@@ -427,39 +427,6 @@ paths:
           description: Missing or invalid content type.
         "500":
           $ref: "#/components/responses/ServiceError"
-  /users/{memberID}/domains:
```

Comment:
Could you update `ListDomains` to reflect the addition of the userID query parameter